test/integration/PooledStaking/rewardsAndBurns.js

```diff
@@ -85,7 +85,7 @@ describe('burns', function () {
       expireTime: '7972408607',
       generationTime: '7972408607001',
       currency,
-      period: 61,
+      period: 63,
```

Comment:
unresolved:  no root cause for why this is an issue. only found to fail for the number 61.